testing/trino-testing/src/main/java/io/trino/testing/TestingConnectorBehavior.java

```diff
@@ -26,6 +26,8 @@
     SUPPORTS_LIMIT_PUSHDOWN,
 
     SUPPORTS_TOPN_PUSHDOWN,
+    // TODO: Remove this once https://github.com/trinodb/trino/issues/7496 is done
```

Comment:
I don't believe this will ever be done for _all_ the connectors, so you can remove the TODO from here.
(add some info in the issue though)